.env

```diff
@@ -50,4 +50,4 @@ _APP_MAINTENANCE_RETENTION_AUDIT=1209600
 _APP_USAGE_STATS=enabled
 _APP_LOGGING_PROVIDER=
 _APP_LOGGING_CONFIG=
-OPENRUNTIMES_INACTIVE_THRESHOLD=60
\ No newline at end of file
+_APP_RUNTIMES_INACTIVE_THRESHOLD=60
```

Comment:
Let's keep it under the `APP_FUNCTIONS` prefix.

Reply:
Alright, done :+1: